reconcile/gitlab_members.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import logging
+
+import utils.gql as gql
+from utils.config import get_config
+from utils.gitlab_api import GitLabApi
+
+GROUPS_QUERY = """
+{
+    groups: gitlabgroups_v1{
```

Comment:
putting this here, but i think we already talked about it -
i think that `gitlabgroups` is not exactly the right name for this...
maybe its actually the representation of a gitlab instance, and we can add that to the schema as well?

we can probably even re-use such an entity in other integrations.
if this object will include the token, we may even be able to remove some configuration from the toml file in a future PR.

if this was too abstract, here is an example:
https://gitlab.cee.redhat.com/service/app-interface/blob/master/data/dependencies/ci-int/ci-int.yml
(ignore the `plugins` stuff)
the `path` in this example is a path to a secret in vault which holds the token.